card/mod/05_standard/set/rstar/rules.rb

```diff
@@ -246,7 +246,7 @@ def default_follow_item_args args
            else
              card.rule_set.follow_label
            end
-    link_target = if card.rule_set.codename == 'self'
+    link_target = if card.rule_set.tag.codename == 'self'
```

Comment:
Wouldn't work: `Card.fetch('Home+*self').right_id` is nil.  I guess because the card is virtual.